Error: Property 'setReplySerializer' does not exist on type 'FastifyInstance'

## 🐛 Bug Report

I'm trying to use setReplySerializer() in my Fastify-powered TypeScript project. However, an error is thrown:

`error TS2339: Property 'setReplySerializer' does not exist on type 'FastifyInstance<Server, IncomingMessage, ServerResponse, FastifyLoggerInstance>'.`

## To Reproduce

Steps to reproduce the behavior:

```js
import * as Fastify from 'fastify';

(async () => {
  try {
    const server = Fastify.fastify();

    // Based upon https://www.fastify.io/docs/master/Server/#setreplyserializer
    server.setReplySerializer(function (payload: any, statusCode: any) {
      return `my serialized ${statusCode} content: ${payload}`;
    });

    await server.listen(3000, '0.0.0.0');
  } catch (err) {
    console.error(err);
  }
})();

```

## Expected behavior

No TypeScript error

## Your Environment

- *Node version*: 12
- *Fastify version*: 3.0.0-rc.4
- *TypeScript version*: 3.9.5
- *os*: Linux

--- a/test/types/instance.test-d.ts
+++ b/test/types/instance.test-d.ts
@@ -33,7 +33,7 @@ server.setErrorHandler(nodeJSErrorHandler)
 function invalidErrorHandler(error: number) {}
 expectError(server.setErrorHandler(invalidErrorHandler))
 
-server.setReplySerializer((function (payload: unknown, statusCode: number) {
+server.setReplySerializer((function (payload, statusCode) {
   expectType<unknown>(payload)
   expectType<number>(statusCode)
   return 'serialized'
@@ -41,3 +41,6 @@ server.setReplySerializer((function (payload: unknown, statusCode: number) {
 
 function invalidReplySerialzer(payload: number, statusCode: string) {}
 expectError(server.setReplySerializer(invalidReplySerialzer))
+
+function serializerWithInvalidReturn(payload: unknown, statusCode: number) {}
+expectError(server.setReplySerializer(serializerWithInvalidReturn))

--- a/test/types/instance.test-d.ts
+++ b/test/types/instance.test-d.ts
@@ -32,3 +32,12 @@ server.setErrorHandler(nodeJSErrorHandler)
 
 function invalidErrorHandler(error: number) {}
 expectError(server.setErrorHandler(invalidErrorHandler))
+
+server.setReplySerializer((function (payload: unknown, statusCode: number) {
+  expectType<unknown>(payload)
+  expectType<number>(statusCode)
+  return 'serialized'
+}))
+
+function invalidReplySerialzer(payload: number, statusCode: string) {}
+expectError(server.setReplySerializer(invalidReplySerialzer))

--- a/types/instance.d.ts
+++ b/types/instance.d.ts
@@ -241,8 +241,8 @@ export interface FastifyInstance<
   setSerializerCompiler(schemaCompiler: FastifySchemaCompiler): FastifyInstance<RawServer, RawRequest, RawReply, Logger>;
 
   /**
- * Set the reply serializer for all routes.
- */
+  * Set the reply serializer for all routes.
+  */
   setReplySerializer(replySerializer: (payload: unknown, statusCode: number) => string): FastifyInstance<RawServer, RawRequest, RawReply, Logger>;
 
   /**

--- a/types/instance.d.ts
+++ b/types/instance.d.ts
@@ -241,6 +241,11 @@ export interface FastifyInstance<
   setSerializerCompiler(schemaCompiler: FastifySchemaCompiler): FastifyInstance<RawServer, RawRequest, RawReply, Logger>;
 
   /**
+ * Set the reply serializer for all routes.
+ */
+  setReplySerializer(replySerializer: (payload: unknown, statusCode: number) => string): FastifyInstance<RawServer, RawRequest, RawReply, Logger>;
+
+  /**
    * Add a content type parser
    */
   addContentTypeParser: AddContentTypeParser<RawServer, RawRequest>;

